client/src/linkManager.js

```diff
@@ -37,16 +38,27 @@ export class linkManager extends LuigiClientBase {
    * @param {Object} modalSettings opens a view in a modal. Use these settings to configure the modal's title and size
    * @param {string} modalSettings.title modal title. By default, it is the node label. If there is no label, it is left empty
    * @param {('l'|'m'|'s')} [modalSettings.size="l"] size of the modal
+   * @param {Object} splitViewSettings opens a view in a split view. Use these settings to configure the split view's behaviour
+   * @param {string} splitViewSettings.title split view title. By default, it is the node label. If there is no label, it is left empty
+   * @param {number} [splitViewSettings.size=40] size of the split view in percent
+   * @param {boolean} [splitViewSettings.collapsed=false] create split view but leave it closed initially
+   * @returns {Object} handle that allows functions to control the splitView: collapse, expand, setSize, onCollapse, onExpand, onResize, exists, getSize, isCollapsed
    * @example
    * LuigiClient.linkManager().navigate('/overview')
    * LuigiClient.linkManager().navigate('users/groups/stakeholders')
    * LuigiClient.linkManager().navigate('/settings', null, true) // preserve view
    */
-  navigate(path, sessionId, preserveView, modalSettings) {
+  navigate(path, sessionId, preserveView, modalSettings, splitViewSettings) {
     if (this.options.errorSkipNavigation) {
       this.options.errorSkipNavigation = false;
       return;
     }
+    if ([modalSettings, splitViewSettings].filter(obj => !!obj).length > 1) {
```

Comment:
Maybe `if (modalSettings && splitViewSettings)` or if you want to be more strict `if (Boolean(modalSettings) && Boolean(splitViewSettings))` do the same and they are more simple ;)

Reply:
done